bones/date.py

```diff
@@ -202,10 +205,94 @@ def serializeForPost(self):
 	def serializeForDocument(self):
 		return self.serializeForPost()
 
+
 def CheckForDateBone(moduleName, boneName, skelStucture, *args, **kwargs):
 	return skelStucture[boneName]["type"] == "date"
 
-#Register this Bone in the global queue
+
+class DateRangeFilterPlugin(html5.Div):
+	def __init__(self, extension, view, modul, *args, **kwargs):
+		super(DateRangeFilterPlugin, self).__init__(*args, **kwargs)
+		self.view = view
+		self.extension = extension
+		self.modul = modul
+		self.mutualExclusiveGroupTarget = "daterange-filter"
+		self.mutualExclusiveGroupKey = extension["target"]
+		self.filterChangedEvent = EventDispatcher("filterChanged")
+		title = html5.H2()
+		title.appendChild(html5.TextNode(extension["name"]))
+		self.appendChild(title)
+
+		# from daterange value
+		self.fromDatepicker = html5.Input()
+		self.fromDatepicker["type"] = "date"
+		self.fromDatepicker["id"] = "from-date-%s" % self.extension["target"]
+		self.fromDatepicker["class"] = "extended-date-range-filter"
+		fromLabel = html5.Label("Startdatum")
```

Comment:
Please provide `translate("begin date")` or something similar and put appropriate translations into translations/de.py